homeassistant/components/webostv/quality_scale.yaml

```diff
@@ -0,0 +1,61 @@
+rules:
+  # Bronze
+  action-setup: todo
+  appropriate-polling: todo
+  brands: done
+  common-modules:
+    status: exempt
+    comment: The integration does not use common patterns.
+  config-flow-test-coverage: todo
```

Comment:
`test_form_abort_uuid_configured` is starting a new flow at L263 - L270, but I am missing what the point of that one is

Reply:
Verify that if SSDP advertise a new host (IP change) we update an existing entry

Reply:
No I mean we start a flow and do nothing with it

Reply:
I missed that, understood, will need to be revised after removing legacy UUID migration code